image_data_guide/01_download_data.ipynb

```diff
@@ -0,0 +1,546 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Clear outputs before you make a merge. We want to have a clean looking output. :)

Reply:
Outputs cleared